Makefile

```diff
@@ -194,8 +194,10 @@ lint:
 	flake8
 
 format-check: uncrustify
-	black --check .
-	isort --check .
+	black --check --diff .
+	isort --check --diff .
+
+Co-authored-by: Brian Cosgrove <bcosgrove@paypal.com>
```

Comment:
These lines seem to have been added by accident
```suggestion
```

Reply:
My mistake.